pkg/cortex/modules.go

```diff
@@ -193,7 +194,24 @@ func (t *Cortex) initQuerier(cfg *Config) (err error) {
 		return
 	}
 
-	queryable, engine := querier.New(cfg.Querier, t.distributor, t.store)
+	var store querier.ChunkStore
+	if cfg.Ingester.V2.Enabled {
```

Comment:
That's a much better name, thank you :)

Reply:
Though I'm unsure if it should be in the storage package, because that's already a sub-package of the chunk package.

Reply:
Right. I believe we should pick the best option from the config file / CLI flags perspective, cause packages can be refactored in subsequent PRs without introducing breaking changes, but the opposite is not true.

Another option may be:
1. Accept to just add `engine` to `storage:`
2. The `tsdb:` is configured through a new root level config block, stored in `pkg/storage/tsdb/` (I've added the `pkg/storage` cause we may evaluate to move `pkg/chunk` there in the future)

For example:
```
storage:
    engine: tsdb

tsdb:
    // config options here
```

Thoughts?

Reply:
Since I know I've been quite picky with my comments about the config, I'm offering you a PR against your feature branch:
https://github.com/thorfour/cortex/pull/1

What's your take?

Reply:
Awesome! Merged it